### PR TITLE
added 1.12.3 stable release notes for d4mac, d4win docs

### PR DESCRIPTION
**Pull request by londoncalling**
### Proposed changes

1.12.3 Stable release notes for Mac and Windows

### Please take a look

@MagnusS @dave-tucker @mstanleyjones 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview processing.

Built with commit d6f8a15cca6df016c7faf39b4c6aaf4f507f6b22

https://app.netlify.com/sites/docker-docs/deploys/5823939d6686744a4054d981

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,71 @@ Release notes for _stable_ and _beta_ releases are listed below. You can learn a
...
+
+- Fixed an issue where Docker for Mac was incorrectly reported as updated
+
+- Channel is now displayed in About box
+
+- Crash reports are sent over Bugsnag rather than HockeyApp
+
+- Fixed an issue where some windows did not claim focus correctly
+
+- Add UI when switching channel to prevent user losing containers and settings
```
Comment (mstanleyjones) [inline docker-for-mac/release-notes.md]: Verb inconsistency between present and past tense ("Fixed" vs "Add")

File: docker-for-mac/release-notes.md [hunk 2]
```diff
@@ -18,6 +18,71 @@ Release notes for _stable_ and _beta_ releases are listed below. You can learn a
...
+
+- Check disk capacity before toolbox import
+
+- Import certificates in `etc/ssl/certs/ca-certificates.crt`
+
+- disk: make the "flush" behaviour configurable for database-like workloads. This works around a performance regression in 1.12.1.
+
+**Networking**
+
+- Fixed application of system or custom proxy settings over container restart
```
Comment (mstanleyjones) [inline docker-for-mac/release-notes.md]: This one doesn't start with a category, maybe like "Proxy:"

File: docker-for-mac/release-notes.md [hunk 3]
```diff
@@ -18,6 +18,71 @@ Release notes for _stable_ and _beta_ releases are listed below. You can learn a
...
+
+- DNS: reduce the number of UDP sockets consumed on the host
+
+- VPNkit: improve the connection-limiting code to avoid running out of sockets on the host
+
+- UDP: handle diagrams bigger than 2035, up to the configured macOS kernel limit
+
+- UDP: make the forwarding more robust; drop packets and continue rather than stopping
+
+**Filesharing**
```
Comment (mstanleyjones) [inline docker-for-mac/release-notes.md]: "Filesharing" or "File sharing"?

**Comment by londoncalling** [comment]
Okay, @mstanleyjones thanks for comments, all addressed. I'm going to merge this.

**Pull request merged**